tesseract/tesseract_urdf/include/tesseract_urdf/box.h

```diff
@@ -88,13 +88,17 @@ inline tesseract_common::StatusCode::Ptr parse(tesseract_geometry::Box::Ptr& box
     return std::make_shared<tesseract_common::StatusCode>(BoxStatusCategory::ErrorAttributeSizeConversion, status_cat);
 
   double l, w, h;
-  if (!tesseract_common::toNumeric<double>(tokens[0], l) || !(l > 0))
+  tesseract_common::toNumeric<double>(tokens[0], l);
+  tesseract_common::toNumeric<double>(tokens[1], w);
+  tesseract_common::toNumeric<double>(tokens[2], h);
+
+  if (!(l > 0))
```

Comment:
Through code coverage I found that no matter what is done it will never get here if they are not already numeric because above I check the token vector using the isNumeric which will always fail there.

Reply:
Perhaps we should put a comment here and in similar spots about that.

Reply:
Done!